packages/element-library/src/elementTypes.js

```diff
@@ -42,6 +43,11 @@ const elementTypes = [
     name: __('Product', 'web-stories'),
     ...productElement,
   },
+  {
+    type: 'audio_sticker',
```

Comment:
Hmm let's rename it to `audioSticker`. We don't typically use snake_case

```suggestion
    type: 'audioSticker',
```

Note: requires some changes in quick actions as well, and anywhere else the element type is used.

Reply:
Apologies! I missed that 😅
Corrected in the upcoming commit and will rectify other, if any.